openassessment/xblock/submission_mixin.py

```diff
@@ -286,6 +286,14 @@ def create_submission(self, student_item_dict, student_sub_data):
 
         return submission
 
+    @XBlock.json_handler
+    def get_student_username(self, data, suffix):  # pylint: disable=unused-argument
```

Comment:
Nit: it would be good to have a new unit test for this handler.

Reply:
I thought that too but I'm not exactly sure how I would meaningfully test it, i feel like any test would mock away basically everything

Reply:
Yeah, that's fine though - you just want something to check that there's a `/get_student_username` endpoint available from this XBlock, and that it returns some username in its data.